src/components/snapshot/overview/SnapshotOverviewPanel.tsx

```diff
@@ -102,6 +102,11 @@ function SnapshotOverviewPanel(props: SnapshotOverviewPanelProps) {
               useMarkdown={true}
             />
           </Grid>
+          <Grid item xs={12}>
+            <Grid item xs={4}>
+              {renderTextFieldValue('DUOS Id', snapshot.duosFirecloudGroup?.duosId)}
```

Comment:
Could we please display this as DUOS ID rather than DUOS Id?

Could we also include a tooltip on an information icon with this text on hover? "Link with a DUOS dataset ID to automatically add DAC approved users as snapshot readers"

Reply:
Yes, can make these changes!